railway deploy fail

@claude railway deploy failed again. here are the build logs: 

2026-04-24T04:50:53.080022307Z [inf]  
[notice] A new release of pip is available: 24.0 -> 26.0.1
[notice] To update, run: /app/venv/bin/python3 -m pip install --upgrade pip

2026-04-24T04:50:54.072911109Z [inf]  Done!

2026-04-24T04:50:54.178861206Z [inf]  sh build.sh
2026-04-24T04:50:58.606223420Z [inf]  copy /app
2026-04-24T04:51:00.694704696Z [inf]  copy /app
2026-04-24T04:51:00.695965561Z [inf]  [railpack] merge ghcr.io/railwayapp/railpack-runtime:mise-2026.3.17, $packages:mise, $build
2026-04-24T04:51:00.838445971Z [inf]  [railpack] merge ghcr.io/railwayapp/railpack-runtime:mise-2026.3.17, $packages:mise, $build
2026-04-24T04:51:21.682210575Z [inf]  importing to docker
2026-04-24T04:51:23.448384033Z [inf]  [auth] sharing credentials for production-us-west2.railway-registry.com
2026-04-24T04:51:23.448415236Z [inf]  [auth] sharing credentials for production-us-west2.railway-registry.com
2026-04-24T04:51:33.171172203Z [inf]  importing to docker
2026-04-24T04:51:55.077702588Z [inf]  [92mBuild time: 103.67 seconds[0m
2026-04-24T04:52:04.587220383Z [inf]  
2026-04-24T04:52:04.587225851Z [inf]  [35m====================
Starting Healthcheck
====================
[0m
2026-04-24T04:52:04.587226988Z [inf]  [37mPath: /api/health[0m
2026-04-24T04:52:04.587228063Z [inf]  [37mRetry window: 5m0s[0m
2026-04-24T04:52:04.587228998Z [inf]  
2026-04-24T04:52:06.506711066Z [inf]  [93mAttempt #1 failed with service unavailable. Continuing to retry for 4m58s[0m
2026-04-24T04:52:09.550207901Z [inf]  [93mAttempt #2 failed with service unavailable. Continuing to retry for 4m55s[0m
2026-04-24T04:52:12.153228741Z [inf]  [93mAttempt #3 failed with service unavailable. Continuing to retry for 4m52s[0m
2026-04-24T04:52:17.999083562Z [inf]  [93mAttempt #4 failed with service unavailable. Continuing to retry for 4m46s[0m
2026-04-24T04:52:26.665747512Z [inf]  [93mAttempt #5 failed with service unavailable. Continuing to retry for 4m37s[0m
2026-04-24T04:52:52.702666273Z [inf]  [93mAttempt #6 failed with service unavailable. Continuing to retry for 4m11s[0m
2026-04-24T04:53:32.746166447Z [inf]  [93mAttempt #7 failed with service unavailable. Continuing to retry for 3m31s[0m
2026-04-24T04:54:12.793051635Z [inf]  [93mAttempt #8 failed with service unavailable. Continuing to retry for 2m51s[0m
2026-04-24T04:54:52.839003043Z [inf]  [93mAttempt #9 failed with service unavailable. Continuing to retry for 2m11s[0m
2026-04-24T04:55:32.883161456Z [inf]  [93mAttempt #10 failed with service unavailable. Continuing to retry for 1m31s[0m
2026-04-24T04:56:13.179284798Z [inf]  [93mAttempt #11 failed with service unavailable. Continuing to retry for 51s[0m
2026-04-24T04:56:53.249906692Z [inf]  
2026-04-24T04:56:53.249910769Z [inf]  [91m1/1 replicas never became healthy![0m
2026-04-24T04:56:53.249911712Z [inf]  [91mHealthcheck failed![0m



here are the deploy logs: 
2026-04-24T04:52:37.077317106Z [err]      import chromadb.utils.embedding_functions as ef
2026-04-24T04:52:37.077320080Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/utils/embedding_functions.py", line 7, in <module>
2026-04-24T04:52:37.077323042Z [err]      from chromadb.api.types import (
2026-04-24T04:52:37.077325986Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/api/types.py", line 102, in <module>
2026-04-24T04:52:37.077329002Z [err]      ImageDType = Union[np.uint, np.int_, np.float_]
2026-04-24T04:52:37.077333694Z [err]                                           ^^^^^^^^^
2026-04-24T04:52:37.077336716Z [err]    File "/app/venv/lib/python3.11/site-packages/numpy/__init__.py", line 778, in __getattr__
2026-04-24T04:52:37.077339507Z [err]      raise AttributeError(
2026-04-24T04:52:37.990381319Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 713, in spawn_worker
2026-04-24T04:52:37.990388745Z [err]      mod = importlib.import_module(module)
2026-04-24T04:52:37.990392691Z [err]      worker.init_process()
2026-04-24T04:52:37.990396078Z [err]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T04:52:37.990400202Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 136, in init_process
2026-04-24T04:52:37.990406720Z [err]      return self.load_wsgiapp()
2026-04-24T04:52:37.990410648Z [err]             ^^^^^^^^^^^^^^^^^^^
2026-04-24T04:52:37.990414753Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 47, in load_wsgiapp
2026-04-24T04:52:37.990423493Z [err]  [2026-04-24 04:52:37 +0000] [7] [ERROR] Exception in worker process
2026-04-24T04:52:37.990426489Z [err]                      ^^^^^^^^^^^
2026-04-24T04:52:37.990429275Z [err]      return util.import_app(self.app_uri)
2026-04-24T04:52:37.990431455Z [err]  Traceback (most recent call last):
2026-04-24T04:52:37.990432808Z [err]      self.load_wsgi()
2026-04-24T04:52:37.990440122Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T04:52:37.990441929Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 57, in load
2026-04-24T04:52:37.990444692Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 148, in load_wsgi
2026-04-24T04:52:37.990451932Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/util.py", line 377, in import_app
2026-04-24T04:52:37.990453788Z [err]      self.wsgi = self.app.wsgi()
2026-04-24T04:52:37.990453855Z [err]                  ^^^^^^^^^^^^^^^
2026-04-24T04:52:37.990461357Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/base.py", line 66, in wsgi
2026-04-24T04:52:37.990465313Z [err]      self.callable = self.load()
2026-04-24T04:52:37.990704043Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/api/__init__.py", line 7, in <module>
2026-04-24T04:52:37.990721294Z [err]    File "/mise/installs/python/3.11.15/lib/python3.11/importlib/__init__.py", line 126, in import_module
2026-04-24T04:52:37.990725435Z [err]      return _bootstrap._gcd_import(name[level:], package, level)
2026-04-24T04:52:37.990730022Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T04:52:37.990735638Z [err]    File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2026-04-24T04:52:37.990739099Z [err]    File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2026-04-24T04:52:37.990742674Z [err]    File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2026-04-24T04:52:37.990746482Z [err]    File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2026-04-24T04:52:37.990750870Z [err]    File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2026-04-24T04:52:37.990755146Z [err]    File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2026-04-24T04:52:37.990758742Z [err]    File "/app/backend/app.py", line 9, in <module>
2026-04-24T04:52:37.990762011Z [err]      from routes import health_bp, employees_bp, time_entries_bp, clock_sessions_bp, users_bp, grok_bp, jobs_bp, timesheet_submissions_bp, pto_bp, availability_bp, shift_swaps_bp
2026-04-24T04:52:37.990765214Z [err]    File "/app/backend/routes/__init__.py", line 6, in <module>
2026-04-24T04:52:37.990768159Z [err]      from .grok import bp as grok_bp
2026-04-24T04:52:37.990771337Z [err]    File "/app/backend/routes/grok.py", line 8, in <module>
2026-04-24T04:52:37.990776181Z [err]      import chromadb
2026-04-24T04:52:37.990780450Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/__init__.py", line 3, in <module>
2026-04-24T04:52:37.990784082Z [err]      from chromadb.api.client import Client as ClientCreator
2026-04-24T04:52:37.992621571Z [err]      from chromadb.api.models.Collection import Collection
2026-04-24T04:52:37.992628049Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/api/models/Collection.py", line 7, in <module>
2026-04-24T04:52:37.992631877Z [err]      import chromadb.utils.embedding_functions as ef
2026-04-24T04:52:37.992635709Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/utils/embedding_functions.py", line 7, in <module>
2026-04-24T04:52:37.992639864Z [err]      from chromadb.api.types import (
2026-04-24T04:52:37.992777458Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/api/types.py", line 102, in <module>
2026-04-24T04:52:37.992781344Z [err]      ImageDType = Union[np.uint, np.int_, np.float_]
2026-04-24T04:52:37.992785251Z [err]                                           ^^^^^^^^^
2026-04-24T04:52:37.992790437Z [err]    File "/app/venv/lib/python3.11/site-packages/numpy/__init__.py", line 778, in __getattr__
2026-04-24T04:52:37.992794155Z [err]      raise AttributeError(
2026-04-24T04:52:37.992797514Z [err]  AttributeError: `np.float_` was removed in the NumPy 2.0 release. Use `np.float64` instead.
2026-04-24T04:52:37.992801197Z [err]  `np.float_` was removed in the NumPy 2.0 release. Use `np.float64` instead.
2026-04-24T04:52:37.992804666Z [err]  [2026-04-24 04:52:37 +0000] [7] [INFO] Worker exiting (pid: 7)
2026-04-24T04:52:37.992808582Z [err]  [2026-04-24 04:52:37 +0000] [8] [ERROR] Exception in worker process
2026-04-24T04:52:37.992813169Z [err]  Traceback (most recent call last):
2026-04-24T04:52:37.992816492Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 713, in spawn_worker
2026-04-24T04:52:37.992819734Z [err]      worker.init_process()
2026-04-24T04:52:37.992823168Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 136, in init_process
2026-04-24T04:52:37.992826301Z [err]      self.load_wsgi()
2026-04-24T04:52:37.994253993Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T04:52:37.994259529Z [err]    File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2026-04-24T04:52:37.994263116Z [err]    File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2026-04-24T04:52:37.994283706Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 148, in load_wsgi
2026-04-24T04:52:37.994291912Z [err]      self.wsgi = self.app.wsgi()
2026-04-24T04:52:37.994296926Z [err]                  ^^^^^^^^^^^^^^^
2026-04-24T04:52:37.994300681Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/base.py", line 66, in wsgi
2026-04-24T04:52:37.994304226Z [err]      self.callable = self.load()
2026-04-24T04:52:37.994307491Z [err]                      ^^^^^^^^^^^
2026-04-24T04:52:37.994310742Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 57, in load
2026-04-24T04:52:37.994313882Z [err]      return self.load_wsgiapp()
2026-04-24T04:52:37.994317058Z [err]             ^^^^^^^^^^^^^^^^^^^
2026-04-24T04:52:37.994321675Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 47, in load_wsgiapp
2026-04-24T04:52:37.994326037Z [err]      return util.import_app(self.app_uri)
2026-04-24T04:52:37.994329222Z [err]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T04:52:37.994332394Z [err]    File "/app/venv/lib/python3.11/site-packages/gunicorn/util.py", line 377, in import_app
2026-04-24T04:52:37.994335602Z [err]      mod = importlib.import_module(module)
2026-04-24T04:52:37.994338856Z [err]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-04-24T04:52:37.994342106Z [err]    File "/mise/installs/python/3.11.15/lib/python3.11/importlib/__init__.py", line 126, in import_module
2026-04-24T04:52:37.994345877Z [err]      return _bootstrap._gcd_import(name[level:], package, level)
2026-04-24T04:52:38.015848774Z [err]    File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2026-04-24T04:52:38.015852177Z [err]    File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2026-04-24T04:52:38.015855837Z [err]    File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2026-04-24T04:52:38.015861690Z [err]    File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2026-04-24T04:52:38.015866541Z [err]    File "/app/backend/app.py", line 9, in <module>
2026-04-24T04:52:38.015870291Z [err]      from routes import health_bp, employees_bp, time_entries_bp, clock_sessions_bp, users_bp, grok_bp, jobs_bp, timesheet_submissions_bp, pto_bp, availability_bp, shift_swaps_bp
2026-04-24T04:52:38.015874402Z [err]    File "/app/backend/routes/__init__.py", line 6, in <module>
2026-04-24T04:52:38.015878287Z [err]      from .grok import bp as grok_bp
2026-04-24T04:52:38.015882253Z [err]    File "/app/backend/routes/grok.py", line 8, in <module>
2026-04-24T04:52:38.015886012Z [err]      import chromadb
2026-04-24T04:52:38.015890089Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/__init__.py", line 3, in <module>
2026-04-24T04:52:38.015893635Z [err]      from chromadb.api.client import Client as ClientCreator
2026-04-24T04:52:38.015896861Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/api/__init__.py", line 7, in <module>
2026-04-24T04:52:38.015899811Z [err]      from chromadb.api.models.Collection import Collection
2026-04-24T04:52:38.015903010Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/api/models/Collection.py", line 7, in <module>
2026-04-24T04:52:38.015906701Z [err]      import chromadb.utils.embedding_functions as ef
2026-04-24T04:52:38.015909944Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/utils/embedding_functions.py", line 7, in <module>
2026-04-24T04:52:38.015912867Z [err]      from chromadb.api.types import (
2026-04-24T04:52:38.017048183Z [err]    File "/app/venv/lib/python3.11/site-packages/chromadb/api/types.py", line 102, in <module>
2026-04-24T04:52:38.017052507Z [err]      ImageDType = Union[np.uint, np.int_, np.float_]
2026-04-24T04:52:38.017056848Z [err]                                           ^^^^^^^^^
2026-04-24T04:52:38.017060172Z [err]    File "/app/venv/lib/python3.11/site-packages/numpy/__init__.py", line 778, in __getattr__
2026-04-24T04:52:38.017063395Z [err]      raise AttributeError(
2026-04-24T04:52:38.017069592Z [err]  AttributeError: `np.float_` was removed in the NumPy 2.0 release. Use `np.float64` instead.
2026-04-24T04:52:38.017073492Z [err]  `np.float_` was removed in the NumPy 2.0 release. Use `np.float64` instead.
2026-04-24T04:52:38.017077056Z [err]  [2026-04-24 04:52:37 +0000] [8] [INFO] Worker exiting (pid: 8)
2026-04-24T04:52:38.017080344Z [err]  [2026-04-24 04:52:37 +0000] [1] [ERROR] Worker (pid:5) exited with code 3.
2026-04-24T04:52:38.017083654Z [err]  [2026-04-24 04:52:37 +0000] [1] [INFO] Worker (pid:6) was sent SIGTERM!
2026-04-24T04:52:38.017087522Z [err]  [2026-04-24 04:52:37 +0000] [1] [INFO] Worker (pid:7) was sent SIGTERM!
2026-04-24T04:52:38.017091315Z [err]  [2026-04-24 04:52:37 +0000] [1] [INFO] Worker (pid:8) was sent SIGTERM!
2026-04-24T04:52:38.071029651Z [err]  [2026-04-24 04:52:38 +0000] [1] [ERROR] Shutting down: Master
2026-04-24T04:52:38.071032994Z [err]  [2026-04-24 04:52:38 +0000] [1] [ERROR] Reason: Worker failed to boot.

--- a/backend/requirements.txt
+++ b/backend/requirements.txt
@@ -4,6 +4,7 @@ flask-limiter>=3.0.0
 psycopg2-binary>=2.9.0
 openai>=1.0.0
 chromadb>=0.4.0,<0.5.0
+numpy<2.0
 pypdf>=4.0.0
 python-docx>=1.1.0
 requests

